### PR TITLE
Added script to create empty file view

### PR DESCRIPTION
**Pull request by sgosline**
Addresses issue #58

### CODE REVIEW

File: bin/create_empty_fileview.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Create empty file view from a json of interest
+
+'''
+
+import os
+import sys
+import json
+import urlparse
+import urllib
+import pandas as pd
```
Comment (kdaily) [inline bin/create_empty_fileview.py]: pandas is not used

**Review by kdaily — COMMENTED** [review]
I'm just realizing now that this functionality looks a lot like [bin/json2schema.py](https://github.com/Sage-Bionetworks/synAnnotationUtils/blob/master/bin/json2schema.py), just to a view schema instead of a table schema.

File: bin/create_empty_fileview.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sgosline) [reply bin/create_empty_fileview.py]: removed

**Review by kdaily — APPROVED** [review]
lg, but lets compare to that other one

**Pull request merged**